Clarify Suspicious AWS Events

commit-dkp opened the pull request:
Rather than define what is unusual, define what is expected.

Commit:
Clarify Suspicious AWS Events

Rather than define what is unusual, define what is expected.

rogeruiz commented on content/docs/ops/maintenance-list.md:
```diff
@@ -5,48 +5,73 @@ menu:
-The platform requires regular support and maintenance activities to remain in a compliant state. If you are on support and you can’t complete any of the items personally, you are responsible for ensuring that an appropriate person does it.
+The platform requires regular support and maintenance activities to remain in a 
+compliant state. If you are on support and you can’t complete any of the items 
+personally, you are responsible for ensuring that an appropriate person does it.
```
Can we mention here that people should reach out at the end of program-level standups in order to get visibility with people who are in the best position to help?

commit-dkp replied:
Added in https://github.com/18F/cg-site/pull/922/commits/96f57d379f819e0af0564b483d9e5d0c7b18600e .

rogeruiz commented on content/docs/ops/maintenance-list.md:
```diff
@@ -5,48 +5,73 @@ menu:
 
-- On each bosh director run `bosh deployments` and verify the stemcell in-use for each deployment is current.
+- On each bosh director run `bosh deployments` and verify the stemcell in-use 
+for each deployment is current.
```
We're talking about having a pipeline task for this. But it shouldn't stop this update. :+1:

for your eyes, https://gsa-tts.slack.com/archives/C0ENP71UG/p1496895499351167

rogeruiz commented on content/docs/ops/maintenance-list.md:
```diff
@@ -61,7 +86,9 @@ Use the [AWS Console](http://docs.aws.amazon.com/govcloud-us/latest/UserGuide/go
 
-If any suspicious activity is discovered discuss with appropriate team members and if necessary follow the [Security Incident Response Guide]({{< relref "docs/ops/security-ir.md" >}}).
+If you observe any non-Terraform activity, discuss the event(s) with the 
+indicated cloud.gov operator(s) and if necessary follow the 
```
Can we make `indicated cloud.gov operator(s)` a link to [this Google doc](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit#gid=0)?

rogeruiz reviewed (changes requested):
This looks great. 👍 here's some changes. Also in the future I think it's good to do whitespace changes to the docs, but it made it difficult to review. I'd say either focus on just the changes and whitespace / line break changes for the lines your editing or parts your editing **or** say in the opening comment that it should be viewed in _Rich Diff Mode_.

Commit:
Link To Operators Doc

We have a Google Doc that lists the cloud.gov operators for the support person to reach out to.  Also, noting ideal forums to reach out to the team if help is required.

rogeruiz reviewed (approved):
:shipit:
